test/relayer_test.go

```diff
@@ -20,5 +20,8 @@ func getTestChainFactory() ibctest.ChainFactory {
 }
 
 func TestRelayer(t *testing.T) {
+	if testing.Short() {
+		return
```

Comment:
The pattern in the Go standard library is to use t.Skip to mark a short test as skipped, as opposed to a return which counts as a success.

Reply:
Ah, good call.